README.md

```diff
@@ -1,2 +1,60 @@
 # admission-trial
- 
+ # Portfolio
+
+This is my Microverse project called "Portfolio" (Mobile and Desktop version) built using HTML and CSS only.
+
+Built with: HTML and CSS
+
+Linters such as :
+
+1. Sytle Lint
+2. WebHint
+3. Lighthouse
+
+# Getting Started
+
+- Create a new repo
+- Create a new branch according to the GitHub flow rules.
+- Add linters
+- Add a git ignore file
+- Add HTML & CSS files
+- Open a pull request
+
+### Prerequisites
+
+- Git. Which is a open source distributed version control system designed for source code management.
+- A text editor (e.g Visual Studio Code, Vim.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+
+### Setup
+
+- Using git, clone the code to your machine or download a Zip File.
```

Comment:
Kindly check how a professional README is constructed. Your `Getting Started`, `Prerequisite` and `Setup` sections should carry your audience along on what is needed to run the application and how to go about setting up a local version.